buildSrc/build.gradle

```diff
@@ -16,7 +16,7 @@ dependencies {
     implementation 'com.github.spotbugs.snom:spotbugs-gradle-plugin:5.1.4'
     implementation 'io.spring.dependency-management:io.spring.dependency-management.gradle.plugin:1.1.3'
     implementation 'org.hidetake.swagger.generator:org.hidetake.swagger.generator.gradle.plugin:2.19.2'
-    implementation 'org.springframework.boot:spring-boot-gradle-plugin:2.7.17'
+    implementation 'org.springframework.boot:spring-boot-gradle-plugin:3.1.2'
     implementation 'bio.terra:terra-test-runner:0.2.0-SNAPSHOT'
```

Comment:
I wonder if we can remove this override/direct dependency now.

https://github.com/DataBiosphere/terra-billing-profile-manager/blob/59bceb86b4f44d38bf71c12e30f6e698d0e716e1/service/build.gradle#L18